src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1190,6 +1203,21 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         }
     }
 
+    bool TermControl::BroadcastKeyEvent(const WORD vkey,

```

Comment:
I'd caution on this naming _in all layers_. This is "a key event was broadcasted directly to me," which really is just closer to "`RawWriteKeyEvent`". Right now it seems like this will _cause_ a broadcast (and the immediate next questions are "to whom? from whom?"

Reply:
I think I'm gonna go with `RawWriteKeyEvent`. `IDirectKeyListener` does need to broadcast up too, so I'm not sure we can just converge these 😕